verilog/tools/kythe/verilog_kythe_extractor.cc

```diff
@@ -73,42 +63,43 @@ static void ExtractKytheFacts(
     if (!f.good()) {
       LOG(FATAL) << "Can't write to " << output_path;
     }
-    f << verilog::kythe::KytheFactsPrinter(facts_tree) << std::endl;
+    f << verilog::kythe::KytheFactsPrinter(file_list_facts_tree) << std::endl;
   }
+
+  return exit_status;
 }
 
 int main(int argc, char** argv) {
   const auto usage =
       absl::StrCat("usage: ", argv[0], " [options] <file> [<file>...]\n",
                    R"(
 verilog_kythe_extractor is a simple command-line utility
-to extract kythe indexing facts from the given file.
+to extract kythe indexing facts from the given file list.
 
-Expected Input: verilog file.
+Expected Input: verilog file list which contains the path of verilog files.
 Expected output: Produces Indexing Facts for kythe.
 
 Example usage:
-verible-verilog-kythe-extractor files...)");
+verible-verilog-kythe-extractor file_list)");
 
   const auto args = verible::InitCommandLine(usage, &argc, &argv);
 
-  int exit_status = 0;
   std::vector<verilog::kythe::IndexingFactNode> indexing_facts_trees;
 
-  // All positional arguments are file names.  Exclude program name.
-  for (const auto filename :
-       verible::make_range(args.begin() + 1, args.end())) {
-    std::string content;
-    if (!verible::file::GetContents(filename, &content).ok()) {
-      exit_status = 1;
-      continue;
-    }
+  std::string content;
+  if (!verible::file::GetContents(args[1], &content).ok()) {
```

Comment:
Just FYI: if you don't have a file list, you can still use shell to create one on-the-fly, using `-` as a filename.  This can be useful for convenient testing.

```shell
v-v-k-extractor - <<EOF
file1.sv
file2.sv
EOF
```